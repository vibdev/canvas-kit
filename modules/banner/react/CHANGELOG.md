# Change Log

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## [0.1.25](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.24...@workday/canvas-kit-react-banner@0.1.25) (2019-06-20)


### Bug Fixes

* broad stroke accessibility fixes ([9803412](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/commits/9803412))





## [0.1.24](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.23...@workday/canvas-kit-react-banner@0.1.24) (2019-06-18)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.23](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.22...@workday/canvas-kit-react-banner@0.1.23) (2019-06-18)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.22](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.21...@workday/canvas-kit-react-banner@0.1.22) (2019-06-17)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.21](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.20...@workday/canvas-kit-react-banner@0.1.21) (2019-06-17)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.20](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.19...@workday/canvas-kit-react-banner@0.1.20) (2019-06-14)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.19](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.18...@workday/canvas-kit-react-banner@0.1.19) (2019-06-13)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.18](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.17...@workday/canvas-kit-react-banner@0.1.18) (2019-06-13)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.17](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.16...@workday/canvas-kit-react-banner@0.1.17) (2019-06-10)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.16](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.15...@workday/canvas-kit-react-banner@0.1.16) (2019-06-05)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.15](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.14...@workday/canvas-kit-react-banner@0.1.15) (2019-06-03)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.14](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.13...@workday/canvas-kit-react-banner@0.1.14) (2019-06-03)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.13](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.12...@workday/canvas-kit-react-banner@0.1.13) (2019-05-31)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.12](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.11...@workday/canvas-kit-react-banner@0.1.12) (2019-05-24)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.11](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.10...@workday/canvas-kit-react-banner@0.1.11) (2019-05-22)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.10](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.9...@workday/canvas-kit-react-banner@0.1.10) (2019-05-21)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.9](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.8...@workday/canvas-kit-react-banner@0.1.9) (2019-05-15)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.8](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.7...@workday/canvas-kit-react-banner@0.1.8) (2019-05-14)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.7](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.6...@workday/canvas-kit-react-banner@0.1.7) (2019-05-10)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.6](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.5...@workday/canvas-kit-react-banner@0.1.6) (2019-05-09)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.5](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.4...@workday/canvas-kit-react-banner@0.1.5) (2019-05-09)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.4](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.3...@workday/canvas-kit-react-banner@0.1.4) (2019-05-09)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.3](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.2...@workday/canvas-kit-react-banner@0.1.3) (2019-05-07)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## [0.1.2](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-banner/compare/@workday/canvas-kit-react-banner@0.1.1...@workday/canvas-kit-react-banner@0.1.2) (2019-05-06)

**Note:** Version bump only for package @workday/canvas-kit-react-banner





## 0.1.1 (2019-04-30)

**Note:** Version bump only for package @workday/canvas-kit-react-banner
