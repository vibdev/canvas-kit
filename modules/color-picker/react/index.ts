import ColorInput from './lib/ColorInput';
import ColorPreview from './lib/ColorPreview';

export default ColorInput;
export {ColorInput, ColorPreview};
export * from './lib/ColorInput';
export * from './lib/ColorPreview';
