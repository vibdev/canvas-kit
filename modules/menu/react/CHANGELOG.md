# Change Log

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## [0.0.21](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-menu/compare/@workday/canvas-kit-react-menu@0.0.20...@workday/canvas-kit-react-menu@0.0.21) (2019-06-13)

**Note:** Version bump only for package @workday/canvas-kit-react-menu





## [0.0.20](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-menu/compare/@workday/canvas-kit-react-menu@0.0.19...@workday/canvas-kit-react-menu@0.0.20) (2019-06-13)

**Note:** Version bump only for package @workday/canvas-kit-react-menu





## [0.0.19](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-menu/compare/@workday/canvas-kit-react-menu@0.0.18...@workday/canvas-kit-react-menu@0.0.19) (2019-06-10)

**Note:** Version bump only for package @workday/canvas-kit-react-menu





## [0.0.18](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-menu/compare/@workday/canvas-kit-react-menu@0.0.17...@workday/canvas-kit-react-menu@0.0.18) (2019-06-05)

**Note:** Version bump only for package @workday/canvas-kit-react-menu





## [0.0.17](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-menu/compare/@workday/canvas-kit-react-menu@0.0.16...@workday/canvas-kit-react-menu@0.0.17) (2019-06-05)

**Note:** Version bump only for package @workday/canvas-kit-react-menu





## [0.0.16](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-menu/compare/@workday/canvas-kit-react-menu@0.0.15...@workday/canvas-kit-react-menu@0.0.16) (2019-06-03)

**Note:** Version bump only for package @workday/canvas-kit-react-menu





## [0.0.15](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-menu/compare/@workday/canvas-kit-react-menu@0.0.14...@workday/canvas-kit-react-menu@0.0.15) (2019-06-03)

**Note:** Version bump only for package @workday/canvas-kit-react-menu





## [0.0.14](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-menu/compare/@workday/canvas-kit-react-menu@0.0.13...@workday/canvas-kit-react-menu@0.0.14) (2019-05-31)

**Note:** Version bump only for package @workday/canvas-kit-react-menu





## [0.0.13](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-menu/compare/@workday/canvas-kit-react-menu@0.0.12...@workday/canvas-kit-react-menu@0.0.13) (2019-05-30)


### Bug Fixes

* **Card:** background color should be white ([31955bd](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-menu/commits/31955bd))





## [0.0.12](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-menu/compare/@workday/canvas-kit-react-menu@0.0.11...@workday/canvas-kit-react-menu@0.0.12) (2019-05-24)

**Note:** Version bump only for package @workday/canvas-kit-react-menu





## [0.0.11](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-menu/compare/@workday/canvas-kit-react-menu@0.0.10...@workday/canvas-kit-react-menu@0.0.11) (2019-05-22)

**Note:** Version bump only for package @workday/canvas-kit-react-menu





## [0.0.10](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-menu/compare/@workday/canvas-kit-react-menu@0.0.9...@workday/canvas-kit-react-menu@0.0.10) (2019-05-21)

**Note:** Version bump only for package @workday/canvas-kit-react-menu





## [0.0.9](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-menu/compare/@workday/canvas-kit-react-menu@0.0.8...@workday/canvas-kit-react-menu@0.0.9) (2019-05-15)

**Note:** Version bump only for package @workday/canvas-kit-react-menu





## [0.0.8](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-menu/compare/@workday/canvas-kit-react-menu@0.0.7...@workday/canvas-kit-react-menu@0.0.8) (2019-05-14)

**Note:** Version bump only for package @workday/canvas-kit-react-menu





## [0.0.7](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-menu/compare/@workday/canvas-kit-react-menu@0.0.6...@workday/canvas-kit-react-menu@0.0.7) (2019-05-10)

**Note:** Version bump only for package @workday/canvas-kit-react-menu





## [0.0.6](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-menu/compare/@workday/canvas-kit-react-menu@0.0.5...@workday/canvas-kit-react-menu@0.0.6) (2019-05-09)

**Note:** Version bump only for package @workday/canvas-kit-react-menu





## [0.0.5](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-menu/compare/@workday/canvas-kit-react-menu@0.0.4...@workday/canvas-kit-react-menu@0.0.5) (2019-05-09)


### Bug Fixes

* **common:** move storybook utils to outside folder ([1f661b6](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-menu/commits/1f661b6))





## [0.0.4](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-menu/compare/@workday/canvas-kit-react-menu@0.0.3...@workday/canvas-kit-react-menu@0.0.4) (2019-05-09)

**Note:** Version bump only for package @workday/canvas-kit-react-menu





## [0.0.3](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-menu/compare/@workday/canvas-kit-react-menu@0.0.2...@workday/canvas-kit-react-menu@0.0.3) (2019-05-07)

**Note:** Version bump only for package @workday/canvas-kit-react-menu





## [0.0.2](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/canvas-kit-react-menu/compare/@workday/canvas-kit-react-menu@0.0.1...@workday/canvas-kit-react-menu@0.0.2) (2019-05-06)

**Note:** Version bump only for package @workday/canvas-kit-react-menu





## 0.0.1 (2019-04-29)

**Note:** Version bump only for package @workday/canvas-kit-react-menu
