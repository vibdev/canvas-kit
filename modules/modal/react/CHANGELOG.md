# Change Log

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## [0.0.2](https://ghe.megaleo.com/design/canvas-kit-react/tree/master/modules/modal/compare/@workday/canvas-kit-react-modal@0.0.1...@workday/canvas-kit-react-modal@0.0.2) (2019-06-20)

**Note:** Version bump only for package @workday/canvas-kit-react-modal





## 0.0.1 (2019-06-20)

**Note:** Version bump only for package @workday/canvas-kit-react-modal
