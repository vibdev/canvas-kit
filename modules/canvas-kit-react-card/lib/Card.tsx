import * as React from 'react';
import styled from 'react-emotion';
import {colors, depth as depthValues, type, spacing} from '@workday/canvas-kit-react-core';
import {CanvasSpacingValue} from '@workday/canvas-space-web';
import {CanvasDepthValue} from '@workday/canvas-depth-web';

export interface CardProps extends React.HTMLAttributes<HTMLDivElement> {
  /**
   * Title of the card
   */
  heading?: React.ReactNode;

  /**
   * Padding within card contents
   */
  padding: 0 | CanvasSpacingValue;

  /**
   * Depth of card
   */
  depth: CanvasDepthValue;
}

const Box = styled('div')<CardProps>(
  {
    border: `1px solid ${colors.soap500}`,
    borderRadius: 3,
  },
  ({depth}) => depth,
  ({padding}) => ({padding})
);

const Header = styled('div')(type.h3, {
  marginBottom: spacing.m,
});

const Body = styled('div')(type.body);

export default class Card extends React.Component<CardProps> {
  public static defaultProps = {
    depth: depthValues[2],
    padding: spacing.l,
  };

  public render() {
    const {heading, padding, ...elemProps} = this.props;

    return (
      <Box padding={padding} {...elemProps}>
        {heading && <Header>{heading}</Header>}
        <Body>{this.props.children}</Body>
      </Box>
    );
  }
}
