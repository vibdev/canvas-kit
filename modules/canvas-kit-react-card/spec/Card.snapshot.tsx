import * as React from 'react';
import Card, {CardSize} from '../lib/Card';
import * as renderer from 'react-test-renderer';

describe('Card Snapshots', () => {
  test('renders as expected', () => {
    const component = renderer.create(<Card>Card</Card>);
    expect(component).toMatchSnapshot();
  });

  test('renders a card with title', () => {
    const component = renderer.create(<Card title="Card Title">Card</Card>);
    expect(component).toMatchSnapshot();
  });

  const cards = Array.from(Array(12).keys()).map(size => (
    <Card title="Title" size={(size + 1) as CardSize} key={size} />
  ));

  cards.forEach((card, i) => {
    test(`renders a card with size ${i + 1}`, () => {
      const component = renderer.create(card);
      expect(component).toMatchSnapshot();
    });
  });

  test(`renders a card with custom breakpoints`, () => {
    const customBreakpoints = {
      sm: 120,
      md: 240,
      lg: 480,
    };

    const component = renderer.create(<Card size={2} breakpoints={customBreakpoints} />);
    expect(component).toMatchSnapshot();
  });
});
