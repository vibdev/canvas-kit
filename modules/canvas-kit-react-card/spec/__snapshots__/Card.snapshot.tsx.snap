// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Snapshots renders a card with custom breakpoints 1`] = `
.emotion-0 {
  font-size: 14px;
  color: #494949;
  font-weight: 400;
  line-height: 20px;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
}

.emotion-1 {
  box-shadow: 0px 4px 8px 0 rgba(0,0,0,0.1);
  border: 1px solid #ced3d9;
  border-radius: 3px;
  padding: 32px;
  width: 100%;
}

@media (min-width:120px) {
  .emotion-1 {
    width: calc(33.333333333333336% - 96px);
  }
}

@media (min-width:240px) {
  .emotion-1 {
    width: calc(16.666666666666668% - 96px);
  }
}

<div
  className="emotion-1"
  size={2}
>
  <div
    className="emotion-0"
  >
    Card
  </div>
</div>
`;

exports[`Card Snapshots renders a card with size 1 1`] = `
.emotion-1 {
  font-size: 14px;
  color: #494949;
  font-weight: 400;
  line-height: 20px;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
}

.emotion-0 {
  font-weight: 700;
  font-size: 20px;
  line-height: 28px;
  color: #323232;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
  margin-bottom: 24px;
}

.emotion-2 {
  box-shadow: 0px 4px 8px 0 rgba(0,0,0,0.1);
  border: 1px solid #ced3d9;
  border-radius: 3px;
  padding: 32px;
  width: 100%;
}

@media (min-width:320px) {
  .emotion-2 {
    width: calc(16.666666666666668% - 96px);
  }
}

@media (min-width:768px) {
  .emotion-2 {
    width: calc(8.333333333333334% - 96px);
  }
}

<div
  className="emotion-2"
  size={1}
>
  <div
    className="emotion-0"
  >
    Title
  </div>
  <div
    className="emotion-1"
  >
    Card
  </div>
</div>
`;

exports[`Card Snapshots renders a card with size 2 1`] = `
.emotion-1 {
  font-size: 14px;
  color: #494949;
  font-weight: 400;
  line-height: 20px;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
}

.emotion-0 {
  font-weight: 700;
  font-size: 20px;
  line-height: 28px;
  color: #323232;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
  margin-bottom: 24px;
}

.emotion-2 {
  box-shadow: 0px 4px 8px 0 rgba(0,0,0,0.1);
  border: 1px solid #ced3d9;
  border-radius: 3px;
  padding: 32px;
  width: 100%;
}

@media (min-width:320px) {
  .emotion-2 {
    width: calc(33.333333333333336% - 96px);
  }
}

@media (min-width:768px) {
  .emotion-2 {
    width: calc(16.666666666666668% - 96px);
  }
}

<div
  className="emotion-2"
  size={2}
>
  <div
    className="emotion-0"
  >
    Title
  </div>
  <div
    className="emotion-1"
  >
    Card
  </div>
</div>
`;

exports[`Card Snapshots renders a card with size 3 1`] = `
.emotion-1 {
  font-size: 14px;
  color: #494949;
  font-weight: 400;
  line-height: 20px;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
}

.emotion-0 {
  font-weight: 700;
  font-size: 20px;
  line-height: 28px;
  color: #323232;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
  margin-bottom: 24px;
}

.emotion-2 {
  box-shadow: 0px 4px 8px 0 rgba(0,0,0,0.1);
  border: 1px solid #ced3d9;
  border-radius: 3px;
  padding: 32px;
  width: 100%;
}

@media (min-width:320px) {
  .emotion-2 {
    width: calc(50% - 96px);
  }
}

@media (min-width:768px) {
  .emotion-2 {
    width: calc(25% - 96px);
  }
}

<div
  className="emotion-2"
  size={3}
>
  <div
    className="emotion-0"
  >
    Title
  </div>
  <div
    className="emotion-1"
  >
    Card
  </div>
</div>
`;

exports[`Card Snapshots renders a card with size 4 1`] = `
.emotion-1 {
  font-size: 14px;
  color: #494949;
  font-weight: 400;
  line-height: 20px;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
}

.emotion-0 {
  font-weight: 700;
  font-size: 20px;
  line-height: 28px;
  color: #323232;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
  margin-bottom: 24px;
}

.emotion-2 {
  box-shadow: 0px 4px 8px 0 rgba(0,0,0,0.1);
  border: 1px solid #ced3d9;
  border-radius: 3px;
  padding: 32px;
  width: 100%;
}

@media (min-width:320px) {
  .emotion-2 {
    width: calc(100% - 96px);
  }
}

@media (min-width:768px) {
  .emotion-2 {
    width: calc(33.333333333333336% - 96px);
  }
}

<div
  className="emotion-2"
  size={4}
>
  <div
    className="emotion-0"
  >
    Title
  </div>
  <div
    className="emotion-1"
  >
    Card
  </div>
</div>
`;

exports[`Card Snapshots renders a card with size 5 1`] = `
.emotion-1 {
  font-size: 14px;
  color: #494949;
  font-weight: 400;
  line-height: 20px;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
}

.emotion-0 {
  font-weight: 700;
  font-size: 20px;
  line-height: 28px;
  color: #323232;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
  margin-bottom: 24px;
}

.emotion-2 {
  box-shadow: 0px 4px 8px 0 rgba(0,0,0,0.1);
  border: 1px solid #ced3d9;
  border-radius: 3px;
  padding: 32px;
  width: 100%;
}

@media (min-width:320px) {
  .emotion-2 {
    width: calc(100% - 96px);
  }
}

@media (min-width:768px) {
  .emotion-2 {
    width: calc(41.66666666666667% - 96px);
  }
}

<div
  className="emotion-2"
  size={5}
>
  <div
    className="emotion-0"
  >
    Title
  </div>
  <div
    className="emotion-1"
  >
    Card
  </div>
</div>
`;

exports[`Card Snapshots renders a card with size 6 1`] = `
.emotion-1 {
  font-size: 14px;
  color: #494949;
  font-weight: 400;
  line-height: 20px;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
}

.emotion-0 {
  font-weight: 700;
  font-size: 20px;
  line-height: 28px;
  color: #323232;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
  margin-bottom: 24px;
}

.emotion-2 {
  box-shadow: 0px 4px 8px 0 rgba(0,0,0,0.1);
  border: 1px solid #ced3d9;
  border-radius: 3px;
  padding: 32px;
  width: 100%;
}

@media (min-width:320px) {
  .emotion-2 {
    width: calc(100% - 96px);
  }
}

@media (min-width:768px) {
  .emotion-2 {
    width: calc(50% - 96px);
  }
}

<div
  className="emotion-2"
  size={6}
>
  <div
    className="emotion-0"
  >
    Title
  </div>
  <div
    className="emotion-1"
  >
    Card
  </div>
</div>
`;

exports[`Card Snapshots renders a card with size 7 1`] = `
.emotion-1 {
  font-size: 14px;
  color: #494949;
  font-weight: 400;
  line-height: 20px;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
}

.emotion-0 {
  font-weight: 700;
  font-size: 20px;
  line-height: 28px;
  color: #323232;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
  margin-bottom: 24px;
}

.emotion-2 {
  box-shadow: 0px 4px 8px 0 rgba(0,0,0,0.1);
  border: 1px solid #ced3d9;
  border-radius: 3px;
  padding: 32px;
  width: 100%;
}

@media (min-width:320px) {
  .emotion-2 {
    width: calc(100% - 96px);
  }
}

@media (min-width:768px) {
  .emotion-2 {
    width: calc(58.333333333333336% - 96px);
  }
}

<div
  className="emotion-2"
  size={7}
>
  <div
    className="emotion-0"
  >
    Title
  </div>
  <div
    className="emotion-1"
  >
    Card
  </div>
</div>
`;

exports[`Card Snapshots renders a card with size 8 1`] = `
.emotion-1 {
  font-size: 14px;
  color: #494949;
  font-weight: 400;
  line-height: 20px;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
}

.emotion-0 {
  font-weight: 700;
  font-size: 20px;
  line-height: 28px;
  color: #323232;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
  margin-bottom: 24px;
}

.emotion-2 {
  box-shadow: 0px 4px 8px 0 rgba(0,0,0,0.1);
  border: 1px solid #ced3d9;
  border-radius: 3px;
  padding: 32px;
  width: 100%;
}

@media (min-width:320px) {
  .emotion-2 {
    width: calc(100% - 96px);
  }
}

@media (min-width:768px) {
  .emotion-2 {
    width: calc(66.66666666666667% - 96px);
  }
}

<div
  className="emotion-2"
  size={8}
>
  <div
    className="emotion-0"
  >
    Title
  </div>
  <div
    className="emotion-1"
  >
    Card
  </div>
</div>
`;

exports[`Card Snapshots renders a card with size 9 1`] = `
.emotion-1 {
  font-size: 14px;
  color: #494949;
  font-weight: 400;
  line-height: 20px;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
}

.emotion-0 {
  font-weight: 700;
  font-size: 20px;
  line-height: 28px;
  color: #323232;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
  margin-bottom: 24px;
}

.emotion-2 {
  box-shadow: 0px 4px 8px 0 rgba(0,0,0,0.1);
  border: 1px solid #ced3d9;
  border-radius: 3px;
  padding: 32px;
  width: 100%;
}

@media (min-width:320px) {
  .emotion-2 {
    width: calc(100% - 96px);
  }
}

@media (min-width:768px) {
  .emotion-2 {
    width: calc(75% - 96px);
  }
}

<div
  className="emotion-2"
  size={9}
>
  <div
    className="emotion-0"
  >
    Title
  </div>
  <div
    className="emotion-1"
  >
    Card
  </div>
</div>
`;

exports[`Card Snapshots renders a card with size 10 1`] = `
.emotion-1 {
  font-size: 14px;
  color: #494949;
  font-weight: 400;
  line-height: 20px;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
}

.emotion-0 {
  font-weight: 700;
  font-size: 20px;
  line-height: 28px;
  color: #323232;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
  margin-bottom: 24px;
}

.emotion-2 {
  box-shadow: 0px 4px 8px 0 rgba(0,0,0,0.1);
  border: 1px solid #ced3d9;
  border-radius: 3px;
  padding: 32px;
  width: 100%;
}

@media (min-width:320px) {
  .emotion-2 {
    width: calc(100% - 96px);
  }
}

@media (min-width:768px) {
  .emotion-2 {
    width: calc(83.33333333333334% - 96px);
  }
}

<div
  className="emotion-2"
  size={10}
>
  <div
    className="emotion-0"
  >
    Title
  </div>
  <div
    className="emotion-1"
  >
    Card
  </div>
</div>
`;

exports[`Card Snapshots renders a card with size 11 1`] = `
.emotion-1 {
  font-size: 14px;
  color: #494949;
  font-weight: 400;
  line-height: 20px;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
}

.emotion-0 {
  font-weight: 700;
  font-size: 20px;
  line-height: 28px;
  color: #323232;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
  margin-bottom: 24px;
}

.emotion-2 {
  box-shadow: 0px 4px 8px 0 rgba(0,0,0,0.1);
  border: 1px solid #ced3d9;
  border-radius: 3px;
  padding: 32px;
  width: 100%;
}

@media (min-width:320px) {
  .emotion-2 {
    width: calc(100% - 96px);
  }
}

@media (min-width:768px) {
  .emotion-2 {
    width: calc(91.66666666666667% - 96px);
  }
}

<div
  className="emotion-2"
  size={11}
>
  <div
    className="emotion-0"
  >
    Title
  </div>
  <div
    className="emotion-1"
  >
    Card
  </div>
</div>
`;

exports[`Card Snapshots renders a card with size 12 1`] = `
.emotion-1 {
  font-size: 14px;
  color: #494949;
  font-weight: 400;
  line-height: 20px;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
}

.emotion-0 {
  font-weight: 700;
  font-size: 20px;
  line-height: 28px;
  color: #323232;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
  margin-bottom: 24px;
}

.emotion-2 {
  box-shadow: 0px 4px 8px 0 rgba(0,0,0,0.1);
  border: 1px solid #ced3d9;
  border-radius: 3px;
  padding: 32px;
  width: 100%;
}

@media (min-width:320px) {
  .emotion-2 {
    width: calc(100% - 96px);
  }
}

@media (min-width:768px) {
  .emotion-2 {
    width: calc(100% - 96px);
  }
}

<div
  className="emotion-2"
  size={12}
>
  <div
    className="emotion-0"
  >
    Title
  </div>
  <div
    className="emotion-1"
  >
    Card
  </div>
</div>
`;

exports[`Card Snapshots renders a card with title 1`] = `
.emotion-2 {
  box-shadow: 0px 4px 8px 0 rgba(0,0,0,0.1);
  border: 1px solid #ced3d9;
  border-radius: 3px;
  padding: 32px;
}

.emotion-1 {
  font-size: 14px;
  color: #494949;
  font-weight: 400;
  line-height: 20px;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
}

.emotion-0 {
  font-weight: 700;
  font-size: 20px;
  line-height: 28px;
  color: #323232;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
  margin-bottom: 24px;
}

<div
  className="emotion-2"
>
  <div
    className="emotion-0"
  >
    Card Title
  </div>
  <div
    className="emotion-1"
  >
    Card
  </div>
</div>
`;

exports[`Card Snapshots renders as expected 1`] = `
.emotion-1 {
  box-shadow: 0px 4px 8px 0 rgba(0,0,0,0.1);
  border: 1px solid #ced3d9;
  border-radius: 3px;
  padding: 32px;
}

.emotion-0 {
  font-size: 14px;
  color: #494949;
  font-weight: 400;
  line-height: 20px;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
}

<div
  className="emotion-1"
>
  <div
    className="emotion-0"
  >
    Card
  </div>
</div>
`;
